<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewPlaylist</class>
 <widget class="QWidget" name="AddNewPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>277</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>277</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new playlist</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>24</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Playlist name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>92</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Author:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>124</y>
     <width>81</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cover:</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>144</y>
     <width>241</width>
     <height>118</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>265</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Load cover image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>314</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>314</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
